sections/dashboard/PortfolioChart/Timeframe.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+
+import { PERIOD_DISPLAY, Period } from 'sdk/constants/period';
+import { setSelectedPortfolioTimeframe } from 'state/futures/reducer';
+import { selectSelectedPortfolioTimeframe } from 'state/futures/selectors';
+import { useAppDispatch, useAppSelector } from 'state/hooks';
+
+const Container = styled.div`
+	background-color: ${(props) => props.theme.colors.selectedTheme.button.fill};
+	display: flex;
+	justify-content: center;
+	align-items: baseline;
+	padding: 3px 5px;
+	gap: 4px;
+	border-radius: 100px;
+`;
+
+const StyledBtn = styled.div<{ isActive: boolean }>`
```

Comment:
@asherism this seems a custom component but can we use one from our library rather than something specific for this chart?

Reply:
You're right it's custom, the only other place we have something like this is the stats page and it has a different design.